### ISSUE TITLE
Feature Request: Multiple hex editor / memory peek windows

### ISSUE THREAD

**Issue by c7fab** [TASEmulators/BizHawk#1334]

**Comment by Asnivor** [comment]
Looks like this is loaded through the tool dialog functions:

https://github.com/TASVideos/BizHawk/blob/master/BizHawk.Client.EmuHawk/tools/ToolManager.cs#L56

As far as I can see, this is designed to ensure that you can never open two of the same tools at the same time.

I wouldn't know where to start implementing a workaround for this, or whether this is actually a good idea in the first place. All this kind of stuff uses the existing cheat system and I suspect the world might end if you had two instances open at once. 

Someone who knows more may be able to clarify though.

**Comment by adelikat** [comment]
Yeah, that's simply not going to happen. A better design would be to show multiple domains at once in a singular hex editor (maybe in a tabular view)
